contracts/src/v0.8/functions/dev/v1_X/accessControl/TermsOfServiceAllowList.sol

```diff
@@ -197,4 +201,17 @@ contract TermsOfServiceAllowList is ITermsOfServiceAllowList, IAccessController,
 
     return blockedSenders;
   }
+
+  function migratePreviouslyAllowedSenders(address[] memory previousSendersToAdd) external override onlyOwner {
```

Comment:
💭 do we want to restrict this to be run only once?

Reply:
I was thinking about something like that as well, but if the diff list gets too large it may need to be run more than once.